mkdocs.yml

```diff
@@ -0,0 +1,75 @@
+site_name: Ethereum plugins documentation
+site_url: https://LedgerHQ.github.io
+site_author: fbeutin@ledger.fr
```

Comment:
SHould be better to have a generic email, like firrmware@... or embedded@... Don't know which ones are available